packages/shared/lib/services/sync/data/records.service.ts

```diff
@@ -33,6 +33,13 @@ export const formatDataRecords = (
     trackDeletes = false,
     softDelete = false
 ): ServiceResponse<SyncDataRecord[]> => {
+    // hashing unique composite key (connection, model, external_id)
+    // to generate stable record ids across script executions
+    const stableId = (rawRecord: DataResponse): string => {
+        const namespace = uuid.v5(`${nango_connection_id}${model}`, uuid.NIL);
+        return uuid.v5(`${nango_connection_id}${model}${rawRecord.id}`, namespace);
```

Comment:
having a namespace hierarchy is not mandatory, we could use a constant root namespace. Let me know what you think?

Reply:
could be just nango_connection_id yes, otherwise I'm fine with this